app/workers/scheduler/feed_cleanup_scheduler.rb

```diff
@@ -6,16 +6,17 @@ class Scheduler::FeedCleanupScheduler
 
   def perform
     redis.pipelined do
-      inactive_users.pluck(:account_id).each do |account_id|
+      inactive_users.each do |account_id|
         redis.del(FeedManager.instance.key(:home, account_id))
+        redis.del(FeedManager.instance.key(:home, account_id, 'reblogs'))
       end
     end
   end
 
   private
 
   def inactive_users
-    User.confirmed.inactive
+    @inactive_users ||= User.confirmed.inactive.pluck(:account_id)
```

Comment:
Style nit that this should maybe become `inactive_user_ids`, but it's only defined in this tiny class, so whatever.

Reply:
Yeah that's uh, not substantial.